src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -165,6 +165,10 @@
           type: string
           short-summary: Load balancer backend pool type.
           long-summary: Load balancer backend pool type, supported values are nodeIP and nodeIPConfiguration.
+        - name: --cluster-service-load-balancer-health-probe-mode
+          type: string
+          short-summary: Set the cluster service health probe mode.
+          long-summary: Set the cluster service health probe mode. Valid values are "Shared" and "Servicenodeport". Default is "Servicenodeport".
```

Comment:
If the param is defined with `arg_type=get_enum_type(health_probe_modes)`, it is not necessary to specify the available values for it
```suggestion
          long-summary: Set the cluster service health probe mode. Default is "Servicenodeport".
```